web/html/src/components/buttons.tsx

```diff
@@ -203,15 +210,20 @@ export class LinkButton extends _ButtonBase<LinkProps> {
           };
     return (
       <a
+        role="button"
```

Comment:
I think this is not correct from an accessibility point of view. At the end of the day, this component works as a link even if it looks like a button, in the sense that if the user clicks it, they will be taken to another page. Setting the role to button contradicts that, so we should not set this value here.